.github/workflows/single-binary.yml

```diff
@@ -175,19 +175,38 @@ jobs:
               memory: "0"
           EOF
           flytectl demo start --image flyte-sandbox-bundled:local --imagePullPolicy Never
-      - name: Register examples
-        uses: unionai/flyte-register-action@v0.0.2
-        with:
-          flytesnacks: true
-          config: boilerplate/flyte/end2end/functional-test-config.yaml
-          project: flytesnacks
-          domain: development
-          version: ${{ env.FLYTESNACKS_VERSION }}
       - name: Install Python dependencies
         run: |
           python -m pip install --upgrade pip
           pip install flytekit
           pip freeze
+      - name: Checkout flytesnacks
+        uses: actions/checkout@v3
+        with:
+          repository: flyteorg/flytesnacks
+          path: flytesnacks
+      - name: Register specific tests
+        run: |
+          for f in basics/basics/basic_workflow.py \
+                  basics/basics/hello_world.py \
+                  basics/basics/lp.py \
+                  basics/basics/named_outputs.py \
+                  control_flow/control_flow/chain_entities.py \
+                  control_flow/control_flow/dynamics.py \
+                  control_flow/control_flow/map_task.py \
+                  control_flow/control_flow/subworkflows.py \
+                  type_system/type_system/custom_objects.py \
+                  type_system/type_system/schema.py \
+                  type_system/type_system/typed_schema.py ;
+          do
+              pyflyte --config ./boilerplate/flyte/end2end/functional-test-config.yaml \
+                  register \
+                  --project flytesnacks \
+                  --domain development \
+                  --image cr.flyte.org/flyteorg/flytekit:py3.9-latest \
```

Comment:
Use latest version of `flytekit` image instead?

Reply:
which image?

Reply:
sync'd offline. Meant latest python version image.